packages/es-components/src/components/containers/tabPanels/Tab.js

```diff
@@ -59,6 +59,7 @@ function Tab({
 }) {
   return (
     <TabButton
+      className={`tab-button-${selected ? 'selected' : 'unselected'}`}
```

Comment:
I think this component should take className as a prop and then suffix or prefix it in the className value you're defining here. This way applications that use this component can still have some control over the styling - they could even use styled-components to control some of that styling.

Reply:
This isn't a big deal at all though. I'll still approve.